eppo-sdk-test/eppo-sdk-test.csproj

```
@@ -1,26 +1,35 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
     <RootNamespace>eppo_sdk_test</RootNamespace>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
+    <OutputType>Library</OutputType>
+    <LangVersion>14</LangVersion>
+    <TargetFramework>net10.0</TargetFramework>
```

I think you'll need to list all supported .net versions here, otherwise the matrix build will error out.
```suggestion
    <TargetFramework>net7.0;net8.0;net10.0</TargetFramework>
```